.github/workflows/publish-python-client.yml

```diff
@@ -19,6 +19,8 @@ jobs:
     steps:
       - name: Download wheels
         run: gh run download ${{ github.event.workflow_run.id }}
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
sorry, i never used this particular 'github.token'. What is this token? Is this the API token running the job? (I'm just wondering if there is any danger exposing this in the environment variables/logs.

Reply:
Ah ok, it seems to be a 1-time token? Seems to be ok then.

Reply:
It is a good point. I added a label in the workflow which ensures the token is only granted read access which should be fine. Here it is only used to download the artifacts from the build workflow

Reply:
Yep, very good.